apollo-router/src/plugins/external.rs

```diff
@@ -413,23 +889,22 @@ fn internalize_header_map(
     Ok(output)
 }
 
-// This macro allows us to use it in our plugin registry!
-// register_plugin takes a group name, and a plugin name.
-//
-// In order to keep the plugin names consistent,
-// we use using the `Reverse domain name notation`
-register_plugin!("experimental", "external", ExternalPlugin);
-
 #[cfg(test)]
 mod tests {
+    use futures::future::BoxFuture;
```

Comment:
could you split the tests in a separate file? This file is getting big

Reply:
fixed in aa0ec75d27d5eae6cdee7a3f2b5f922ee26ee785